libcudacxx/examples/concurrent_hash_table.cu

```diff
@@ -3,128 +3,117 @@
 // Released under the Apache License v2.0 with LLVM Exceptions.
 // See https://llvm.org/LICENSE.txt for license information.
 
-#include <cuda/std/cstdint>
 #include <cuda/std/atomic>
+#include <cuda/std/cstdint>
 
 // TODO: It would be great if this example could NOT depend on Thrust.
-#include <thrust/pair.h>
-#include <thrust/functional.h>
 #include <thrust/allocate_unique.h>
-#include <thrust/host_vector.h>
 #include <thrust/device_vector.h>
+#include <thrust/functional.h>
+#include <thrust/host_vector.h>
 #include <thrust/iterator/counting_iterator.h>
+#include <thrust/pair.h>
 
 #include <cassert>
-#include <random>
-
-#include <iostream>
 #include <cstdio>
-#include <cassert>
+#include <iostream>
+#include <random>
 
 // TODO: This should be upstreamed and then removed.
-namespace thrust {
+namespace thrust
+{
 
 using universal_raw_memory_resource =
-  thrust::system::cuda::detail::cuda_memory_resource<
-    thrust::system::cuda::detail::cudaMallocManaged, cudaFree, void*
-  >;
+  thrust::system::cuda::detail::cuda_memory_resource< thrust::system::cuda::detail::cudaMallocManaged, cudaFree, void* >;
 
 template <typename T>
-using universal_allocator =
-  thrust::mr::stateless_resource_allocator<T, universal_raw_memory_resource>;
+using universal_allocator = thrust::mr::stateless_resource_allocator<T, universal_raw_memory_resource>;
 
 template <typename T>
 using universal_vector = thrust::device_vector<T, universal_allocator<T>>;
 
-} // thrust
+} // namespace thrust
 
-template <
-  typename Key, typename Value,
-  typename Hash     = thrust::identity<Key>,
-  typename KeyEqual = thrust::equal_to<Key>,
-  typename MemoryResource = thrust::universal_raw_memory_resource
->
-struct concurrent_hash_table {
+template < typename Key,
+           typename Value,
+           typename Hash           = thrust::identity<Key>,
+           typename KeyEqual       = thrust::equal_to<Key>,
+           typename MemoryResource = thrust::universal_raw_memory_resource >
```

Comment:
FYI, the CUB PR removed the extra spaces in places like this. It had to be done manually.
```suggestion
template <typename Key,
          typename Value,
          typename Hash           = thrust::identity<Key>,
          typename KeyEqual       = thrust::equal_to<Key>,
          typename MemoryResource = thrust::universal_raw_memory_resource>
```

Reply:
We should just set `SpacesInAngles` to `Never` so that the formatter does the job for us